src/electron/views/device-connect-view/components/android-setup/device-description.scss

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+@import '../../../../../common/styles/fonts.scss';
+
+.content {
+    display: flex;
+    line-height: 19px;
+    margin-top: 24px;
+    margin-bottom: 24px;
+}
+
+.iconContent {
```

Comment:
nit: class names are usually `kebab-cased` (the build process will normalize it to `camelCase` for typescript to use)

Reply:
Seems weird to have different names in the style and in the code, and looking at the existing code I don't see it used a whole lot. Let's discuss this, and maybe make it a point for an engineering discussion.

Reply:
changed